manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/plugins/SecretsDataHandler.java

```diff
@@ -44,7 +43,8 @@
 
     @Override
     public List<ManifestSnippet> handle(ServiceSpec serviceSpec, ManifestContext manifestContext) throws HyscaleException {
-        Secrets secrets = SecretsProvider.getSecrets(serviceSpec);
+     //   Secrets secrets = SecretsProvider.getSecrets(serviceSpec);
+        Secrets secrets = serviceSpec.get(HyscaleSpecFields.secrets, Secrets.class);
```

Comment:
Secrets have to be retrieved from Provider because it can have either a list / key-value pairs and we cannot deserialise directly to list/map . Can you cross-check once about this change ?